mobile/src/main/java/ch/epfl/sdp/mobile/ui/social/SocialScreen.kt

```diff
@@ -142,14 +143,18 @@ fun EmptySearch(modifier: Modifier = Modifier) {
  * @param modifier the [Modifier] for the composable
  */
 @Composable
-fun SearchResultList(players: List<Person>, modifier: Modifier = Modifier) {
+fun SearchResultList(
+    players: List<Person>,
+    modifier: Modifier = Modifier,
+    onClick: (p: Person) -> Unit
```

Comment:
I suggest changing the order of the parameters. In general, you want to keep the trailing lambda notation for `@Composable` content functions.

```suggestion
    players: List<Person>,
    onClick: (Person) -> Unit,
    modifier: Modifier = Modifier,
```